cassandra-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraExecutor.java

```diff
@@ -358,8 +409,34 @@ private CassandraNodeHealthCheckDetails performHealthCheck(@NotNull final Cassan
         return builder.build();
     }
 
-    private static CassandraNodeInfo buildInfo(JmxConnect jmx) throws UnknownHostException {
-        Nodetool nodetool = new Nodetool(jmx);
+    private void processAlive(ExecutorDriver driver) {
```

Comment:
Yep. I've changed the method name and added some javadoc. It's a bit difficult (for me?) to find an expressive method name. (commit 79313aa)